### ISSUE TITLE
Any way to make this work with the non steam version of dsx?

### ISSUE THREAD

**Issue by Pikachetah** [josealissonbr/DualSenseAT#18]
i don't have the steam version of dsx and when it's opened it doesn't recognize it

**Comment by thedig123** [comment]
I dont think so as the steam version as a multitude of updates, if you can swing it I would recommend getting dsx on steam as when it goes to 3.0 you will get a free upgrade to it
